eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -152,14 +149,17 @@ jobs:
   - pwsh: |
       $baselineDir = "$(baselineBenchmarkResultsDownloadDir)"
       $fileNamePattern = "*_$(functionApp).json"
-      $baselineFile = Get-ChildItem -Path $baselineDir -Filter $fileNamePattern | Select-Object -First 1
 
-      if ($baselineFile) {
-        Write-Host "Found baseline benchmark result file: $($baselineFile.FullName)"
-        Write-Host "##vso[task.setvariable variable=baselineBenchmarkResultFilePath]$($baselineFile.FullName)"
-      } else {
-        Write-Host "No baseline benchmark result file found."
+      if (Test-Path $baselineDir -PathType Container) {
+          $baselineFile = Get-ChildItem -Path $baselineDir -Filter $fileNamePattern | Select-Object -First 1
+
+          if ($baselineFile) {
+              Write-Host "Found baseline benchmark result file: $($baselineFile.FullName)"
+              Write-Host "##vso[task.setvariable variable=baselineBenchmarkResultFilePath]$($baselineFile.FullName)"
+          }
       }
+
+      Write-Host "No baseline benchmark result file found."
```

Comment:
Looks like this will currently print even if baseline file is found above. Need either a condition here or a `return` above to short-circuit.

Reply:
Updated it to print more specific error message and only when the file / artifact directory not found.